packages/react-utilities/src/compose/getSlots.ts

```diff
@@ -69,7 +69,7 @@ function getSlot<R extends SlotPropsRecord, K extends keyof R>(
   state: ComponentState<R>,
   slotName: K,
 ): readonly [React.ElementType<R[K]>, R[K]] {
-  if (state[slotName] === undefined) {
+  if (state[slotName] === undefined || state[slotName] === null) {
```

Comment:
If `resolveShorthand()` will not return `null` we can revert this change.

Reply:
Per https://github.com/microsoft/fluentui/pull/21518#discussion_r798294923, I'm going to leave it returning `null` and leave this in.

Reply:
Removed in https://github.com/microsoft/fluentui/commit/13a8d1df32759fa29fcf989ea1d222b6ea13edd1